src/m365/planner/commands/roster/roster-member-remove.spec.ts

```diff
@@ -0,0 +1,303 @@
+    assert(promptIssued);
+  });
+
+  it('prompts before removing the last roster member when confirm option not passed', async () => {
+    let secondPromptOptions: any;
+    sinonUtil.restore(Cli.prompt);
+    sinon.stub(Cli, 'prompt').callsFake(async (options: any) => {
+      if (options.message === `Are you sure you want to remove member '${validUserId}'?`) {
+        return (
+          { continue: true }
+        );
+      }
+      else {
+        secondPromptOptions = options;
+        return (
+          { continue: false }
+        );
+      }
+    });
+
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/planner/rosters/${validRosterId}/members?$select=Id`) {
+        return (singleRosterMemberResponse);
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        rosterId: validRosterId,
+        userId: validUserId
+      }
+    });
+
+    let promptIssued = false;
+
+    if (secondPromptOptions && secondPromptOptions.type === 'confirm' && secondPromptOptions.message === `Are you sure you want to remove the last member from the roster '${validRosterId}'? When the last user is removed, the Roster and all its contents will be deleted within 30 days`) {
+      promptIssued = true;
+    }
+
+    assert(promptIssued);
+  });
+
+  it('aborts removing the specified roster member when confirm option not passed and prompt not confirmed', async () => {
+    const postSpy = sinon.spy(request, 'delete');
+
+    sinonUtil.restore(Cli.prompt);
+    sinon.stub(Cli, 'prompt').callsFake(async () => (
+      { continue: false }
+    ));
+    await command.action(logger, {
+      options: {
+        rosterId: validRosterId,
+        userid: validUserId
+      }
+    });
+    assert(postSpy.notCalled);
+  });
+
+  it('removes the last specified roster member when prompt confirmed', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users?$filter=userPrincipalName eq '${formatting.encodeQueryParameter(validUserName)}'&$select=Id`) {
+        return userResponse;
+      }
+
+      if (opts.url === `https://graph.microsoft.com/beta/planner/rosters/${validRosterId}/members?$select=Id`) {
+        return singleRosterMemberResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    sinon.stub(request, 'delete').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/planner/rosters/${validRosterId}/members/${validUserId}`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    sinonUtil.restore(Cli.prompt);
+    sinon.stub(Cli, 'prompt').callsFake(async () => (
+      { continue: true }
+    ));
+
+    await command.action(logger, {
+      options: {
+        verbose: true,
+        rosterId: validRosterId,
+        userName: validUserName
+      }
+    });
+    assert(loggerLogToStderrSpy.called);
```

Comment:
Now we are only checking whether we logged a verbose message. Let's check if the delete request was actually fired.